omnisafe/envs/custom_env.py

```diff
@@ -0,0 +1,197 @@
+# Copyright 2024 OmniSafe Team. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Example and template for environment customization."""
```

Comment:
We now have examples in omnisafe/examples and tutorials, so why do we still need an example here?

Reply:
This is an example that shows how to create an environment under `omnisafe/envs` and define the environment file. **If users want their environment to be one of the officially supported environments by OmniSafe**, they can access it through this interface. Thank you for the reminder; I will provide more detailed instructions.